### PR TITLE
Task#0

### PR DESCRIPTION
**Pull request by pavelborisofff**
Task#0

### CODE REVIEW

File: app/fastapi_proxy.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ async def get_site(request: Request, path: str = '/'):
         response = await client.get(f'{URL}/{path}', params=request.query_params)
```
Comment (fallncrlss) [inline app/fastapi_proxy.py:15]: Probably, naming of the constant like `BASE_DOMAIN`/`BASE_URL` will be a bit better fit

**Review by fallncrlss — COMMENTED** [review]
Good job!
The solution is made in the right way with the minimal amount of the code, without PEP8 violations, using the proper libraries, so cool👍

File: app/fastapi_proxy.py (hunk 1 above)
Comment (pavelborisofff) [reply app/fastapi_proxy.py:15]: Agree

**Comment by pavelborisofff** [comment]
> Good job!
> The solution is made in the right way with the minimal amount of the code, without PEP8 violations, using the proper libraries, so cool👍

But there is some bugs with digits and  the text with digits like 123456™ and urls' text like github™.com 
It needs to fix logic and make few additional checking. I'll do it next 2-3 days, would it be ok?

**Pull request merged**